testutil/testnet/network.go

```diff
@@ -44,9 +44,9 @@ func NewNetwork(nVals int, createCometStarter func(int) *CometStarter) (Nodes, e
 				return
 			}
 
-			// Notify that the new node's switch is available,
-			// so this node can be peered with the other nodes.
-			switchCh <- n.PEXReactor().Switch
+			// Notify that the new node's switch is available, so this node can be
+			// peered with the other nodes.
+			switchCh <- n.Switch()
```

Comment:
made me laugh, one api is private but a lower layer is still public

Reply:
lol smh